Source/WebKit/UIProcess/XR/openxr/OpenXRLayer.cpp

```diff
@@ -121,6 +212,7 @@ std::optional<PlatformXR::FrameData::ExternalTexture> OpenXRLayer::exportOpenXRT
         .modifier = modifier,
     };
 }
+#endif // OS(ANDROID)
```

Comment:
Same about the comment, either omit or use `!OS(ANDROID)` because this is the end of the non android block.